nextpnr.dockerfile

```diff
@@ -1,8 +1,10 @@
 # Authors:
 #   Anton Blanchard
 #   Unai Martinez-Corral
+#   Sebastian Birke
 #
 # Copyright 2019-2021 Unai Martinez-Corral <unai.martinezcorral@ehu.eus>
+# Copyright 2021      Sebastian Birke      <git@se-bi.de>
```

Comment:
Please, remove this line. It's ok to add the e-mail in line 4 if you wish.

Reply:
Ok, got it now :see_no_evil: 

just pushed again.

Reply:
No worries! It took me some time to understand too. And each open source project is slightly different in this regard, so you need to "learn" it again rather frequently 😄